### PR TITLE
chore: docker (protobuf), typos, links and others

### PR DESCRIPTION
**Pull request by Kaspiman**
## Summary by CodeRabbit

- **New Features**
	- Added a `max_queue_size` configuration parameter to the HTTP settings to control the internal request queue size.
	- Introduced a new method `callIgnoreResponse` in the AsyncRPCInterface for better RPC handling.

- **Bug Fixes**
	- Corrected grammatical errors in documentation for community plugins and middleware references for improved clarity.

- **Documentation**
	- Updated various documentation files for clarity, including phrasing adjustments and capitalization standardizations.

- **Chores**
	- Enhanced Dockerfile configurations for PHP applications to include additional extensions and streamline commands.
	- Updated the PHP environment to version `8.3-alpine` and the RoadRunner base image to version `2024` for improved stability and performance.
	- Removed unnecessary configuration parameters related to relay timeouts across multiple settings.

### CODE REVIEW

File: app-server/nginx+RR/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
-FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/roadrunner-server/roadrunner:latest as roadrunner
-FROM --platform=${TARGETPLATFORM:-linux/amd64} php:8.1-alpine
+FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/roadrunner-server/roadrunner:2024 as roadrunner
+FROM --platform=${TARGETPLATFORM:-linux/amd64} php:8.3-alpine
```
Comment (rustatian) [inline app-server/nginx+RR/Dockerfile:2]: I think we can safely use `latest` here. Just don't want to change the version every year 😄

Comment (rustatian) [reply app-server/nginx+RR/Dockerfile:2]: `latest` itself is not a bad practice, it's just a tag. Bad practice is to use `latest` in production, and that's it.
Let's keep it clear, I do want to update docs, and I'm doing that regularly, the problem is, that that's not possible to maintain a version in one place, like some constant. Thus, please, use `latest` tag.

Comment (Kaspiman) [reply app-server/nginx+RR/Dockerfile]: Done.

File: app-server/nginx-with-rr.md [hunk 2]
```diff
@@ -157,21 +157,18 @@ In this example, we will demonstrate how to use RoadRunner with Nginx in a Docke
 
 ```docker
 FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/roadrunner-server/roadrunner:latest as roadrunner
```
Comment (rustatian) [inline app-server/nginx-with-rr.md:159]: You may add a warning here about the latest tag with:
{% hint style="warning" %}
Consider using the direct version in production. `latest` image tag should be used in the development environments only. (something like this).
{% endhint %}

Comment (Kaspiman) [reply app-server/nginx-with-rr.md:159]: No, that's wrong. Development environments should be the same as production.

**Comment by rustatian** [comment]
Thank you @Kaspiman 👍

**Pull request merged**